docs/data-management.rst

```diff
@@ -207,41 +312,80 @@ If you are an owner of the data, you can move the data between the groups you ar
 
-#. A message ``Checking which linked objects will be moved`` will appear and a spinner to the left of it. Wait until the spinner vanishes and a list of objects to be moved and a list of objects which are not included in the move appears.
+#. A message ``Checking which linked objects will be moved``
+   will appear and a spinner to the left of it. Wait until the spinner
+   vanishes and a list of objects to be moved and a list of objects
+   which are not included in the move appears.
 
    |image5|
 
-#. Check both lists. Please read the note above about which objects are typically not included and reconsider the ``Move`` action again. The ``not included`` objects will not be linked to the ``Moved`` objects anymore if you go ahead with the move, the linkage will be lost.
+#. Check both lists. Please read the note above about which objects are
+   typically not included and reconsider the ``Move`` action.
+   The ``not included`` objects will not be linked to the ``Moved``
+   objects anymore if you go ahead with the move, the linkage will be lost.
 
-#. In case you are happy with the ``Move`` action to go ahead, select a target Dataset or Project or create a new one and click ``OK``.
+#. In case you are happy with the ``Move`` action to go ahead,
+   select a target Dataset or Project or create a new one and click ``OK``.
 
 Move data between groups: administrators
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The administrators can move the data to any group, not only to the group where the owner of the data is a member. Note though that it is not desirable to create a situation where the data belong to someone who is not a member of the group where the data reside.
+The administrators can move the data to any group,
+not only to the group where the owner of the data is a member. 
+Note though that it is not desirable to create a situation where
+the data belong
+to someone who is not a member of the group where the data reside.
 
-Typically an administrator works on behalf of other users in a group where the administrator is not a member. For these cases, some features of OMERO.web help to facilitate the moving of data for others (note that these features are not yet available in the Command Line Interface).
+Typically an administrator works on behalf of other users in a group
+where the administrator is not a member.
+For these cases, some features of OMERO.web help to facilitate the moving
+of data for others (note that these features are
+not yet available in the Command Line Interface).
 
 #. Navigate to the data of a user in a group that you are not a member of.
 
 #. Select the data in the left-hand tree.
 
 #. Right-click and select ``Move to Group...``.
 
-#. Follow further the steps described in the section ``Move data between groups: owners of data`` above, taking note of the ``Not included`` objects.
+#. Follow further the steps described in the section
+   ``Move data between groups: owners of data`` above, taking note of the
```

Comment:
`Move data between groups: owners of data`: Can you turn this into a link to the relevant part of this page?

Reply:
I am struggling with linking to an arbitrary sub-header in the page, except for explicitly linking to an "external" link. The anchor system seems to tell me off if I put an reference to an anchor such as 

 :ref:`Duplicate feature<Duplicatecli>`.

pointing to 

.._Duplicatecli:

anchor in case this is not a chapter header. Not sure how to reference a truly arbitrary place in the text, cc @sbesson

Reply:
All good, fixed in https://github.com/ome/omero-guide-introduction/pull/17/commits/14a13c0ee6cb0a60891f85d9dc6f6c00d40cac3c cc @sbesson